inventory_tools/www/bulk-order.html

```diff
@@ -0,0 +1,46 @@
+{% extends "templates/web.html" %}
+
+{% block page_content %}
+
+<script src="bulk-order.js"></script>
+
+<div class="text-center login-content container">
+    <h3 class="form-signin-heading mt-4 mb-4">Bulk Order</h3>
+    <div style="display: block;" id="line-by-line-order">
+        <div class="input-row-container">
+            <div class="form-group data-row row ml-5 grid-field input-row">
+                <input type="text" id="sku_1" class="form-control col col-xs-6" placeholder="SKU 1" required autofocus>
+                &nbsp;
+                <input type="Number" id="qty_1" min="0" class="form-control col col-xs-4" placeholder="Quantity 1"
+                    required>
+            </div>
+        </div>
+        <button class="btn btn-sm mt-2 btn-secondary grid-add-row w-40" onclick="addRow()">Add Row</button>&nbsp;
+        <button class="btn btn-primary mt-2 btn-sm btn-complete-signup w-60" onclick="createQuote()">
+            Add to Quote</button>
+    </div>
+
+    <div id="bulk-order" style="display: none;">
+        <div class="input-row-container">
+            <div class="form-group data-row row ml-5 grid-field">
+                <textarea id="bulk_paste" class="form-control col col-xs-11" placeholder="Paste your bulk order here..."
+                    required>Paste your bulk order here...</textarea>
+            </div>
+        </div>
+        <button class="btn btn-primary mt-2 btn-sm btn-complete-signup w-60" onclick="createQuote(1)">
```

Comment:
Just for slightly better readability:

```suggestion
        <button class="btn btn-primary mt-2 btn-sm btn-complete-signup w-60" onclick="createQuote(true)">
```